server/index.js

```diff
@@ -91,6 +91,7 @@ app.put('/reviews/:review_id/:type', (req, res) => {
 /* Q&A POST/PUT HANDLERS */
 
 app.post('/qa/questions', (req, res) => {
+  req.body.product_id = Number(req.body.product_id);
```

Comment:
Changing product_id to a number might not be necessary if its value is the same as app.state.currentProductId.